app/styles/dark-theme-variables.less

```diff
@@ -254,3 +255,12 @@
 
 @code-bg-color: @lightred;
 @code-color:    @darkred;
+
+//
+// Console Results Colourisation
+// ---------------------------------------------------
+
+@numeric-result-color: #a0a1ff;
```

Comment:
all the colors , appart from the grays :( are defined in the App Colors section with color names to make them reusable and easily identifiable, also so that we don't have too many shades of the same color 

https://github.com/crate/crate-admin/blob/master/app/styles/light-theme-variables.less#L33